src/Microsoft.DotNet.Build.Tasks/Targets/depProj.targets

```diff
@@ -0,0 +1,74 @@
+
+This file defines the steps in the standard build process specific for NuGet deployment
+projects. The remainder of the build process is defined in Microsoft.Common.targets, 
+which is imported by this file.
+
+Copyright (c) Microsoft. All rights reserved.
+Licensed under the MIT license. See LICENSE file in the project root for full license information.
+***********************************************************************************************
+-->
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Deployment project
+       Restores NuGet dependencies and copies them to the output directory.
+       
+       NuGetTargetMoniker - determined by the TargetFramework* and TargetPlatform* 
+                            properties of the project, can be overidden.
+       NuGetRuntimeIdentifier - defaults to win-$(PlatformTarget), can be overidden
+       NuGetDeploySourceItem - defaults to ReferenceCopyLocalPaths, can be overidden to
+                               specify Reference (for compile assets) or Analyzer(for
+                               analyzer assets)
+                         
+       For the appropriate behavior of P2P references the project should set the 
+       TargetName and TargetExt to match one of the files that will be copied
+       from the packages.
+  -->
+  
+  <PropertyGroup>
+    <NuGetDeploySourceItem Condition="'$(NuGetDeploySourceItem)' == ''">ReferenceCopyLocalPaths</NuGetDeploySourceItem>
+
+    <!-- suppress the attempt to copy build output. -->
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+  </PropertyGroup>
+  
+  <Target Name="CoreCompile">
+
+    <Error Condition="'$(NuGetDeploySourceItem)' != 'ReferenceCopyLocalPaths' AND
+                      '$(NuGetDeploySourceItem)' != 'Reference' AND
+                      '$(NuGetDeploySourceItem)' != 'Analyzer'"
+           Text="Unexpected value for NuGetDeploySourceItem:'$(NuGetDeploySourceItem)'.  Expected ReferenceCopyLocalPaths, Reference, or Analyzer." />
+
+    <ItemGroup>
+      <!-- Don't set IntermediateAssembly since this is not produced -->
+      <IntermediateAssembly Remove="@(IntermediateAssembly)" />
+      
+      <!-- filter to items that came from packages -->
+      <NuGetDeploy Include="@($(NuGetDeploySourceItem))" Condition="'%($(NuGetDeploySourceItem).NuGetPackageId)' != ''" />
+      
+      <!-- remove all existing items from NuGet packages we'll be defining these in our own item -->
+      <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.NuGetPackageId)' != ''"/>
+      <Reference Remove="@(Reference)" Condition="'%(Reference.NuGetPackageId)' != ''"/>
+      <Analyzer Remove="@(Analyzer)" Condition="'%(Analyzer.NuGetPackageId)' != ''"/>
+
+      <!-- add items defined by NuGetDeployItem property to ReferenceCopyLocalPaths -->
+      <ReferenceCopyLocalPaths Include="@(NuGetDeploy)" />
```

Comment:
ReferenceCopyLocalPaths is already populated by the resolution target and common targets will copy it to the output.  If we didn't clear ReferenceCopyLocalPaths we'd binclash with the resolved copy-local runtime assets.  
I could leave Reference and Analyzers defined, but those have no use for a 'depProj' (and references causes RAR to do extra work) so I've cleared them.